services/common/src/components/forms/FormWrapper.tsx

```diff
@@ -85,11 +91,15 @@ const FormWrapper: FC<FormWrapperProps & InjectedFormProps<any>> = ({
     isEditMode,
     isModal,
     formName: props.name,
-    onReset
+    onReset,
   };
-  console.log('open form', props.name)
+  console.log("open form", props.name);
   const dispatch = useDispatch();
-  const formErrors = useSelector(getFormSubmitErrors(props.name));
+
+  const formErrors = useSelector(
+    (state) => getFormSubmitErrors(props.name)(state),
```

Comment:
does this still work if it's just getFormSubmitErrors(props.name) with the state param being implicitly passed in? Just thinking in case we want to update the selector at some point.

Reply:
Ah, this actually didn't yield any real benefit.  I'm going to revert it.